README.md

```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
 Ensure you have Java installed on your machine (preferably Java 8 or later). You can check this by running java -version in your command prompt or terminal.
 
 
-# 1. Installation & setup
+Installation & setup
 
-## Running a local Kafka Environment (optional)
+## 1. Running a local Kafka Environment (optional)
 
-See https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine
+To run a local Docker based Kafka environment, run through the steps documented in the 
+[CLI quickstart](https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine).
 
 Note - the docker environment uses the `kafka_network`
 
-Login to Github Container Registry
-```bash
-% docker login ghcr.io -u <<username>>
->> <<token>>
-``` 
+## 2. Pull the CLI image (optional)
 
-Optional - pull the image
-```bash
-% docker pull ghcr.io/specmesh/specmesh-build-cli 
-sing default tag: latest
-latest: Pulling from specmesh/specmesh-build-cli
-128d54f2c9b1: Pull complete 
-<snip>
-``` 
- 
+> docker pull ghcr.io/specmesh/specmesh-build-cli 
 
 # Steps
 
 ## 1. Checkout this repository
 
-`https://github.com/specmesh/getting-started-apachekafka.git`
+Checkout a local copy of this quickstart repository, e.g. using either over http
 
+> git clone https://github.com/specmesh/getting-started-apachekafka.git
 
-## 2. Create or modify a SpecMesh spec file `acme_simple_range_life_enhancer-api.yaml` 
+or over ssh
 
-Links: 
-- [resources/acme_simple_range_life_enhancer-api.yaml](resources/acme_simple_range_life_enhancer-api.yaml)
-- Short version [resources/snippet-api.yaml](resources/snippet-api.yaml)
```

Comment:
I've removed this short version - it's just confusing. 

Go with one spec - as full / short as you want, but having two is confusing to the user.